src/getUri.js

```diff
@@ -0,0 +1,34 @@
+/*jslint node: true */
+"use strict";
+
+var utils = require("../utils");
+var log = require("npmlog");
+
+module.exports = function(defaultFuncs, api, ctx) {
+  return function getUri(uri, callback) {
+    if(!callback) callback = function() {};
```

Comment:
Can you do [this](https://github.com/Schmavery/facebook-chat-api/blob/3dd39a2c0ae0b20c1202d02f4b70740789f28ccb/src/getUserId.js#L22) instead. It doesn't make much sense to not crash when you don't give a callback.